### DISCUSSION TITLE
novelWriter 1.0 is top story on HN front page. Join to Discussion!

### DISCUSSION THREAD

**Discussion by ghost** [vkbo/novelWriter#567] [General]
- HN thread: https://news.ycombinator.com/item?id=25833328

![HN 2021-1-19 14:00UTC](https://i.imgur.com/ZxLvLoo.png)

**Comment by vkbo** [comment]
Thanks for letting me know. I was wondering what was going on!

**Reply by ghost** [reply]
Some question to You asked by HN users:

- https://news.ycombinator.com/item?id=25833328#25833594
  - Is it suitable for non-technical writers? is there any feedback on them baing able to use markdown, `@pov` tags etc?
  - Did you try some commercial competitors? How would you compare? I understand scrivener is one popular option

**Reply by vkbo** [reply]
* "non-technical" as in the person or the writing itself? I deliberately chose not to write a WYSIWYG editor in order to keep the files plain text and as simple as possible. I've written a WYSIWYG editor before, and didn't like the result. I guess having to use some markup makes it slightly technical in use, but I've also added menu entries and keyboard shortcuts for all the features, so it shouldn't be too difficult to get started.

* I've used it, yes. It's excellent, and perfect if you want a full WYSIWYG editor with a lot of features. I went in a different direction exactly because I wanted something simpler. Also, as a Linux user, the lack of decent options was a factor.
